### PR TITLE
WIP:Add Portsecurity Example

### PR DESCRIPTION
**Pull request by dschwabeS11**
How to allow e.g. hosts from a VPN subnet to talk via an instance port to other machines in openstack.

### CODE REVIEW

File: german/docs/documentation/portsecurity.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# SysEleven Stack Port Security
+
+[TOC]
+
+## Overview
+
+OpenStack uses ports to connect cloud instances to networks and corresponding (virtual) network devices like routers, firewalls.
+
+The port security is quite restrictive since OpenStack is supposed to be an mostly independent system that hosts multiple projects, customers and cloud instances with customized security requirements. Therefor we have to change the port security settings to allow special port customization in our environment. OpenStack provides us with the means to manage the security for each port itself.
```
Comment (stefreak) [inline german/docs/documentation/portsecurity.md]: I'd just say "
By default SysEleven Stack port security will be enforced, which means that
- Incoming traffic that does not originate from your project is not allowed (the `default` security group)
- Only traffic from and to the IP addresses known by OpenStack will be allowed

It is possible to change these restrictions using security groups and the port security settings.

In this article we will focus on the port security settings; for security groups see also <Link>
```

Something like that? what do you think danilo?

**Comment by stefreak** [comment]
@dschwabeS11 why did you open another pull request?

**Comment by dschwabeS11** [comment]
Because I messed up the branch and had to make a clean pull from master. This was one of the "bad habbit" branch that I have had still laying around. Clean merge done via the [other pull request](https://github.com/syseleven/syselevenstack-documentation/pull/63). Not so nice but merge worked properly this way. :-/

**Comment by stefreak** [comment]
@dschwabeS11 if you messed up, you can also do e.g. `git reset --hard <commit> && git push -f`

then the branch is reset to the point in history that you wanted

No need to create pull request spam for this :)

**Comment by stefreak** [comment]
if you want to reset to master, then it would be `git reset --hard origin/master` ....

**Comment by stefreak** [comment]
if you want to reset soft (leaving the changes in your branches as-is, ready to be committed again) then you do `git reset --soft`

**Pull request closed without merging**